patterns/1-initial/good-first-project.md

```diff
@@ -0,0 +1,91 @@
+    - _Ease of contribution_ -> Are the required ancillary resources set up and is the code modular enough to facilitate contribution?
+3. __People Compatibility__
+    - _Team Readiness for InnerSource_ -> Are the team members aware of and ready for the challenges of maintaining an InnerSource project?
+    - _Management Support_ -> Is the management willing to support the developers working on the project and the initiative in general?
+
+## Reference Implementation
+
+The [InnerSource Project Fitness Tracker](https://github.com/tapjdey/InnerSource_Project_Fitness) is one specific implementation of this pattern. It helps estimating the _fitness_ of a candidate project along the six dimensions explained above, and presents an aggregate score which is the weighted sum of the scores for these dimensions. (As an side: Weights are determined by how many people considered a characteristic important in the [InnerSource Survey 2020](https://tapjdey.github.io/InnerSource_Survey_2020/index.html)).
+
+## Resulting Context
+
+In general, application of the framework should help assess the InnerSource-readiness of individual projects.
+
+The _fitness score_ of a project can be used as an indication about its suitability for InnerSource style development, which can help in selecting the most appropriate candidate projects. By choosing projects with higher chances of InnerSource success, you also increase the chances of the overall InnerSource program to be successful.
+
+The questionnaire presented in the tool also serves as a de-facto guide as to which aspects need attention for making a project better suited for InnerSource style development.
+
+## Rationale
+
+The two most important works in this context are by [Klaas-Jan Stol and Brian Fitzgerald](https://ulir.ul.ie/bitstream/handle/10344/4443/Stol_2014_inner.pdf?sequence=2) and [Silona Bonewald](https://innersourcecommons.org/assets/files/InnerSourceChecklist.pdf).
+
+In their work, _Stol and Fitzgerald_ presented a set of nine key factors that are important for organizations to consider when adopting inner source. However, that model looked at an organization as a whole, and while a number of organizational characteristics are applicable to an individual project, some aspects are not, and some project characteristics are unaccounted for in the model. They presented three main factors: Software Product, Practices and Tools, People and Management, and three sub-factors on each of them.
+
+_Silona Bonewald_, on the other hand, presented a near-exhaustive checklist about different aspects to consider while adopting InnerSource at an organization. However, not all of them are applicable while considering an individual project. Ideas from the two works described earlier are combined in framing the pattern presented here.
+
+## Demo
+
+A demo video showing the operation of the above-mentioned tool with a specific implementation of the proposed model is available [here](https://imgur.com/a/wO4EyJj).
+
```

Comment:
```suggestion
```

Removed it here and moved a shortened sentence into the Reference Implementation above.